types/office-js/index.d.ts

```diff
@@ -41232,37 +42990,55 @@ declare namespace Excel {
     enum CalculationType {
         /**
          * Recalculates all cells that Excel has marked as dirty, that is, dependents of volatile or changed data, and cells programmatically marked as dirty.
+         * @remarks
+         * [Api set: ExcelApi 1.1]
          */
         recalculate = "Recalculate",
         /**
          * This will mark all cells as dirty and then recalculate them.
+         * @remarks
+         * [Api set: ExcelApi 1.1]
          */
         full = "Full",
         /**
          * This will rebuild the full dependency chain, mark all cells as dirty and then recalculate them.
+         * @remarks
+         * [Api set: ExcelApi 1.1]
          */
         fullRebuild = "FullRebuild"
     }
     /**
      * @remarks
-     * [Api set: ExcelApi 1.1 for All/Formats/Contents, 1.7 for Hyperlinks & HyperlinksAndFormats.]
+     * [Api set: ExcelApi 1.1 for All, Formats, and Contents. 1.7 for Hyperlinks and RemoveHyperlinks.]
```

Comment:
Now that we have detailed information about the individual fields, we should consider if these types of comments in the API set are necessary.